crates/nu-command/tests/commands/math/round.rs

```diff
@@ -14,8 +14,38 @@ fn can_round_very_large_numbers() {
 fn can_round_very_large_numbers_with_precision() {
     let actual = nu!(
         cwd: ".",
-        "echo 18.13725447800741422899276654867720121457878988 | math round -p 10"
+        "18.13725447800741422899276654867720121457878988 | math round -p 10"
     );
 
     assert_eq!(actual.out, "18.137254478")
 }
+
+#[test]
+fn can_round_integer_with_negative_precision() {
+    let actual = nu!(
+        cwd: ".",
+        "123 | math round -p -1"
+    );
```

Comment:
For tests that don't work with files you can omit the `cwd: "."` Leaving it in doesn't cause a direct problem, so just a style suggestion to help with #9021 et al.

```suggestion
    let actual = nu!(
        "123 | math round -p -1"
    );
```

(may need a rustfmt run afterwards)